Description: Always use the pm-utils backend for now
 Redirecting the suspend / hibernate request to logind requires systemd
 to be running, as logind tries to start the suspend.target /
 hibernate.target via the D-Bus interface provided by systemd.
 The availability of logind does not necessarily imply that systemd is
 available since we made logind D-Bus activatable under sysvinit.
 Also, the systemd version currently in unstable (v44) is too old and
 doesn't provide that functionality yet.
 Ubuntu ships a very basic implementation of that D-Bus inferface in
 systemd-shim which can be run under sysvinit. But we don't have that
 package in Debian (yet).
 For now, it's the safest option to just continue using the old
 pm-utils backend always.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=718493
diff --git a/src/linux/up-backend.c b/src/linux/up-backend.c
index b9f75f5..595a1b2 100644
--- a/src/linux/up-backend.c
+++ b/src/linux/up-backend.c
@@ -51,7 +51,7 @@
 
 #ifdef ENABLE_DEPRECATED
 
-#define LOGIND_AVAILABLE() (access("/run/systemd/seats/", F_OK) >= 0)
+#define LOGIND_AVAILABLE() (FALSE)
 
 #define SD_HIBERNATE_COMMAND	"gdbus call --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 --method org.freedesktop.login1.Manager.Hibernate 'true'"
 #define SD_SUSPEND_COMMAND	"gdbus call --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 --method org.freedesktop.login1.Manager.Suspend 'true'"
